src/components/key.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import calculate from '../logic/calculate';
+
+class key extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(obj) {
+    this.props.handleClick(obj);
+  }
+
+  onClick = () => this.handleClick(calculate(this.props.Result, this.props.buttonName));
+
+  render() {
+    return (
+      <div
+        className={this.props.css}
+        onClick={this.onClick}
+        aria-hidden="true"
+      >
+        {this.props.buttonName}
+      </div>
```

Comment:
- [ ] Kindly fix the error in the calculator, when I enter the numbers in the calculator they are not displayed on the screen, however, produce the correct result.

https://user-images.githubusercontent.com/70269507/207680688-fdbeb84d-8ec9-4715-a083-24311d76e1a2.mov